locksmith/src/controllers/v2/ticketsController.ts

```diff
@@ -67,10 +71,27 @@ export class TicketsController {
 
   async sendEmail(request: Request, response: Response) {
     try {
+      const lockAddress = Normalizer.ethereumAddress(request.params.lockAddress)
       const network = Number(request.params.network)
       const tokenId = request.params.tokenId.toLowerCase()
 
-      await notifyNewKeyToWedlocks(tokenId, network)
+      const keyOwner = await this.web3Service.ownerOf(
```

Comment:
Let's move to middleware so we are using a consistent approach across all controllers please